### PR TITLE
Speech bubbles

### PR DESCRIPTION
**Pull request by Kaes3kuch3n**

### CODE REVIEW

File: Assets/Scripts/Editor/RangeDrawer.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+using UnityEditor;
+using UnityEngine;
+using Utils;
+
+namespace Editor
+{
+    [CustomPropertyDrawer(typeof(Range))]
+    public class RangeDrawer : PropertyDrawer
```
Comment (DavidPgl) [inline Assets/Scripts/Editor/RangeDrawer.cs:8]: Nice

File: Assets/Scripts/UI/SpeechBubble.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        /// <summary>
+        /// Updates the position of the speech bubble depending on the camera distance to the player
+        /// </summary>
+        private void UpdatePosition(float cameraDistance)
+        {
+            float normalizedCameraDistance = (cameraDistance - playerController.CameraDistanceRange.min) /
+                                             (playerController.CameraDistanceRange.max - playerController.CameraDistanceRange.min);
+            Vector3 position = speechBubbleContent.transform.localPosition;
+            position.y = normalizedCameraDistance * translationFactor;
+            speechBubbleContent.transform.localPosition = position;
```
Comment (DavidPgl) [inline Assets/Scripts/UI/SpeechBubble.cs:56]: Worldspace didn't work?

**Review by DavidPgl — CHANGES_REQUESTED** [review]

File: Assets/Scripts/UI/SpeechBubble.cs (hunk 2 above)
Comment (Kaes3kuch3n) [reply Assets/Scripts/UI/SpeechBubble.cs:56]: No, for some reason it didn’t move it along the y-axis in world space but along a combination of all three axis

**Review by DavidPgl — DISMISSED** [review]

**Review by SamuelllG — DISMISSED** [review]

**Comment by SamuelllG** [comment]
Merge conflicts :)

**Comment by Kaes3kuch3n** [comment]
Fixed conflicts

**Review by DavidPgl — APPROVED** [review]

**Review by SamuelllG — APPROVED** [review]

**Pull request merged**